tests/ctst/features/notifications.feature

```diff
@@ -0,0 +1,173 @@
+Feature: Bucket notifications
+    In order to receive notifications
+    As an Artesca User
+    I want to activate notifications
+    And to subscribe to events I want to be notified on
+    And to receive notifications on buckets/objects activities I have subscribed to
+
+  @2.5.0
+  @PreMerge
```

Comment:
You can use a `@BucketNotification` tag if you want, it will ease the filtering if we want to add more tests in the future in multiple jobs where not all tests are expected to run.

Reply:
should we not be able (as a "core" feature of CTST) to filter tests by Feature?

(aka it seems redundant to specify a tag matching the name of the feature for *all* scenarios of a feature)